drop-boxes/register-fasta-dropbox/register-fasta.py

```diff
@@ -65,7 +62,7 @@ def process(transaction):
         project = identifier[:5]
         parentCode = identifier[:10]
     else:
-        print "The identifier "+identifier+" did not match the pattern Q[A-Z]{4}\d{3}\w{2} or checksum"
+        print("The identifier "+identifier+" did not match the pattern Q[A-Z]{4}\d{3}\w{2} or checksum")
```

Comment:
does this work in this version/make a difference?

Reply:
My IDE probably added the parenthesis. I could not exactly test it so I do not know.